HelloWorld/HelloWorld.Service.OnFunctions/Functions/HelloWorld.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using FunctionsHost;
+
+
+namespace HelloWorld.Service.OnFunctions
+{
+    public static class HelloWorld
+    {
+        [FunctionName("HelloWorld")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "post", Route = null)] HttpRequest req,
```

Comment:
It looks like you made the ```Initialize``` function no longer anonymous, but shouldn't the HTTP trigger for the request be anonymous?